CHANGELOG.md

```diff
@@ -1,4 +1,61 @@
-## 2.0.0
+# Optimizely Java X SDK Changelog
+## 2.0.0 Beta
+September 29, 2017
+
+This release is a beta release supporting feature flags and rollouts.
+
+### New Features
+#### Feature Flag Accessors
+You can now use feature flags in the Java SDK. You can experiment on features and rollout features through the Optimizely UI.
+
+- `isFeatureEnabled`
+- `getFeatureVariableBoolean`
+- `getFeatureVariableDouble`
+- `getFeatureVariableInteger`
+- `getFeatureVariableString`
+
+### Breaking Changes
```

Comment:
Please mention here that we are dropping support for Java6

Reply:
https://github.com/optimizely/java-sdk/pull/148